content/reference/sql/version-control/dolt-sql-functions.md

```diff
@@ -349,7 +368,100 @@ With result of single row:
 +------------+-----------------+------------+--------------+---------------+-------------+---------------+----------------+---------------+---------------+----------------+----------------+
 ```
 
-# Version Control Functions 
+## `DOLT_LOG()`
+
+The `DOLT_LOG` table function gets the commit log for all commits reachable from the
+provided revision's `HEAD` (or the current `HEAD` if no revisions provided). `DOLT_LOG()`
+works like [CLI `dolt log` command](../../cli.md#dolt-log).
+
+Note that the `DOLT_LOG()` table function currently has restrictions on how it can be used
+in queries. It does not support aliasing or joining with other tables, and argument values
+must be literal values.
+
+### Privileges
+
+`DOLT_LOG()` table function requires `SELECT` privilege for all tables.
+
+### Options
+
+```sql
+DOLT_LOG(<optional_revision>, <optional_revision>)
+```
+
+The `DOLT_LOG()` table function takes up to two optional revision arguments:
+
+- `optional_revision` — all commits reachable from the revision `HEAD`. If you'd like to
```

Comment:
"all commits reachable from the revision `HEAD`" reads funny to me. Maybe something like:

`optional_revision` – a branch name, tag, or commit ref (with or without an ancestor spec) that specifies which ancestor commits to include in the results. If no revisions are specified, the default is the current branch `HEAD`.